### PR TITLE
Replace travis with GitHub actions

### PR DESCRIPTION
**Pull request by anbraten**
Replaced travis CI with github actions. (Fixes exoframejs/exoframe#282)

Added two workflows:
- `Test`: Lint & Test on every PR and branch
- `Deploy`: Build server, deploy coverage, npm package and docker image on `develop` branch and on tags / releases

Please add needed secrets: 
- `DOCKERHUB_USERNAME`: docker hub username
- `DOCKERHUB_PASSWORD`: docker hub accesstoken
- `NPM_TOKEN`: npm accesstoken

Workflows need to be merged to default branch (`master`) before they will be initially executed.

### CODE REVIEW

File: src/docker/templates/compose.js [hunk 1]
```diff
@@ -144,7 +144,7 @@ exports.executeTemplate = async ({
 
   // re-build images if needed
   const {code: buildExitCode, log: buildLog} = await executeCompose({
-    cmd: ['--project-name', baseName, 'build'],
+    cmd: ['--project-name', baseName, '--force-rm', 'build'],
```
Comment (anbraten) [inline src/docker/templates/compose.js]: Fixed removal of failed docker build containers

**Comment by yamalight** [comment]
@anbraten this looks great! thank you so much! I'll merge over the next few days once I have time to create new tokens ❤️

**Comment by yamalight** [comment]
Alright, secrets added, merging this. Thanks once again for your work! ❤️

**Comment by yamalight** [comment]
@anbraten seems like compose tests fail after merge, any ideas why that might happen? 🤔

**Pull request merged**